EventFilter/L1TRawToDigi/python/gtStage2Raw_cfi.py

```diff
@@ -12,7 +12,7 @@
     TauInputTag    = cms.InputTag("simCaloStage2Digis"),
     JetInputTag    = cms.InputTag("simCaloStage2Digis"),
     EtSumInputTag  = cms.InputTag("simCaloStage2Digis"),
-    EtSumZDCInputTag  = cms.InputTag("simCaloStage2Digis"),
```

Comment:
Yes, using simCaloStage2Digis for the EtSumZDCInputTag should result in no information being passed to the ZDCPacker since ZDC EtSums are not present in the simCaloStage2Digis. I'm not sure what the default value should be since at the time I tested the ZDC Packer in March there was no ZDC simulation so I had to test by comparing the Packer by comparing Unpacked digis (Raw To Digi ) with Unpacked Repacked and Unpacked digis ( Raw to Digi to Raw to Digi) . The test script for that process is [here](https://github.com/missirol/cmssw/blob/devel_l1tZDCEtSums2/EventFilter/L1TRawToDigi/test/test_RawToDigiToRawToDigi_GT.py) .

Reply:
The ZDC emulator that got written was replaced in https://github.com/cms-sw/cmssw/pull/42818 to come from HCAL trigger primitives. At a best guess, the ZDC sums should be pulled from `simHcalTriggerPrimitiveDigis`